transport/src/test/java/io/netty5/channel/local/LocalChannelTest.java

```diff
@@ -223,8 +223,7 @@ public void initChannel(LocalChannel ch) throws Exception {
                 // the ClosedChannelException has been created by AbstractUnsafe rather than transport implementations.
                 if (e.getStackTrace().length > 0) {
                     assertThat(
-                            e.getStackTrace()[0].getClassName(), is(AbstractChannel.class.getName() +
-                                    "$AbstractUnsafe"));
+                            e.getStackTrace()[0].getClassName(), is(AbstractChannel.class.getName()));
                     e.printStackTrace();
```

Comment:
Comment above is outdated, now. Also, I wonder if we can do this check (that no write attempt was made) in a more reliable way? It's pretty weak to base this on just the tip of the stack trace.